docs/source/jit_language_reference.rst

```diff
@@ -131,6 +131,7 @@ These types and features from the :mod:`typing` module are unavailble in TorchSc
    ":any:`typing.Any`", ":any:`typing.Any` is currently in development but not yet released"
    ":any:`typing.NoReturn`", "Not implemented"
    ":any:`typing.Union`", "Unlikely to be implemented (however :any:`typing.Optional` is supported)"
+   ":any:`typing.Sequence`", "Not implemented"
```

Comment:
could you also modify line 66 to remove the ellipsis so it is less confusing? thanks.

Reply:
I've replaced the original text by `Tuple[T0, T1, ..., TN]` instead of removing the ellipsis. What do you think?